src/yuzu/bootmanager.cpp

```diff
@@ -603,19 +604,34 @@ bool GRenderWindow::LoadOpenGL() {
     auto context = CreateSharedContext();
     auto scope = context->Acquire();
     if (!gladLoadGL()) {
+        QMessageBox::critical(
```

Comment:
```suggestion
        QMessageBox::warning(
```

Since this is an error that is safely handled and doesn't threaten the stability of the emulator.

Reply:
> Since this is an error that is safely handled and doesn't threaten the stability of the emulator.

In `GRenderWindow::GetUnsupportedGLExtensions()`, should I change that to `LOG_WARNING` as well?

Reply:
I would use LOG_ERROR it if it needs to stand out. `LOG_CRITICAL` should be reserved for, "Your application is probably going to crash after this occurring" scenarios—basically anything that threatens the entire stability of the application.